x-pack/plugins/uptime/public/components/overview/kuery_bar/kuery_bar.tsx

```diff
@@ -50,11 +50,12 @@ export function KueryBar({
     suggestions: [],
     isLoadingIndexPattern: true,
   });
+  const [suggestionLimit, setSuggestionLimit] = useState(15);
   const [isLoadingSuggestions, setIsLoadingSuggestions] = useState<boolean>(false);
   let currentRequestCheck: string;
 
   const [getUrlParams, updateUrlParams] = useUrlParams();
-  const { search: kuery } = getUrlParams();
+  const { search: kuery, dateRangeStart, dateRangeEnd } = getUrlParams();
```

Comment:
Is this state available in the redux store? This solution won't work for the KB when editing alerts because there won't be any date values in the URL. Ideally we would decouple this component from the URL state values entirely. If this can't work in this PR, perhaps we should extract the date range to props and settle the differences at the implementation points for now.

The alerting KB could take the alert params values for its props and the one on the overview page can keep using URL values for now until we resolve https://github.com/elastic/uptime/issues/218.

Reply:
if there aren't any values in the URL, hook will provide default values now and now-15m, that part is already implemented in hook